expr/functions/integralByInterval/function_test.go

```diff
@@ -0,0 +1,46 @@
+package integralByInterval
+
+import (
+	"testing"
+
+	"github.com/bookingcom/carbonapi/expr/helper"
+	"github.com/bookingcom/carbonapi/expr/metadata"
+	"github.com/bookingcom/carbonapi/expr/types"
+	"github.com/bookingcom/carbonapi/pkg/parser"
+	th "github.com/bookingcom/carbonapi/tests"
+)
+
+func init() {
+	md := New("")
+	evaluator := th.EvaluatorFromFunc(md[0].F)
+	metadata.SetEvaluator(evaluator)
+	helper.SetEvaluator(evaluator)
+	for _, m := range md {
+		metadata.RegisterFunction(m.Name, m.F)
+	}
+}
+
+func TestFunction(t *testing.T) {
+	tests := []th.EvalTestItem{
+		{
+			"integralByInterval(10s,'10s')",
+			map[parser.MetricRequest][]*types.MetricData{
+				{"10s", 0, 1}: {
```

Comment:
Could you please add key names to the struct inits?

Reply:
ok